test/Controllers/ServicesTest.php

```diff
@@ -44,11 +43,10 @@ private function serviceBody($type) {
             case 'create':
                 return new CreateServiceBody(
                     "description", // REQUIRED
-                    15, // duration REQUIRED XXX 0 not accepted but NULL turns into 0
-                    // XXX No max_duration in SDK
```

Comment:
See, this is why I :heart: specs 🤓 

Somehow `:max_duration` was dropped in a prior commit on a couple of the request bodies.  I've fixed in v0.0.9 #21 

Please pull latest version and retry.